spark/src/main/scala/org/apache/mahout/sparkbindings/drm/CheckpointedDrmSpark.scala

```diff
@@ -165,7 +168,14 @@ class CheckpointedDrmSpark[K: ClassTag](
       else if (classOf[Writable].isAssignableFrom(ktag.runtimeClass)) (x: K) => x.asInstanceOf[Writable]
       else throw new IllegalArgumentException("Do not know how to convert class tag %s to Writable.".format(ktag))
 
-    rdd.saveAsSequenceFile(path)
```

Comment:
So are you suggesting that we deprecate dfsWrite? Meaning leave it as is with a deprecated Spark function for now and replace it wherever it's called? Then when Spark removes the deprecated function we remove dfsWrite?

Can we handle your code with an implicit conversion that will get called before dfsWrite?